src/ServiceManagement/Common/Commands.ScenarioTest/Commands.ScenarioTest.csproj

```diff
@@ -224,6 +224,13 @@
     <None Include="Resources\DiagnosticsExtension\Files\diagnostics.wadcfgx">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="Resources\ServiceManagement\Files\dSMSTest.cscfg">
+      <SubType>Designer</SubType>
```

Comment:
you don't need this line <subType>..., do you?

Reply:
No, I think Visual Studio added it automatically.
I can remove it.